### PR TITLE
Cerestation add

### PR DESCRIPTION
**Pull request by frontline03**
# About The Pull Request
Adds cerestation, a map I worked on for skyrat previously but never got added, I fixed all the incompatibilities (I hope) and added it for this code

Areas.DMI and it's corresponding code in Modular_Skyrat updated.
Addition of Cerestation Files
Addition of Cerestation Skyrat files
Maps.txt Updated
Shuttles.DM updated.
## Why It's Good For The Game

Adds a new High-Pop map for the station replace cogstation
image

## A Port?
Original TG pr
https://github.com/tgstation/tgstation/pull/24665

My skyrat PR
https://github.com/Skyrat-SS13/Skyrat13/pull/3051

## Changelog

:cl:
add: Adds Cerestation to main server lineup
/:cl:

### CODE REVIEW

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -620,6 +620,10 @@
 	suffix = "delta"
 	name = "cargo ferry (Delta)"
 
+/datum/map_template/shuttle/cargo/cere
+    suffix = "cere"
+    name = "supply shuttle (cere)"
+
```
Comment (BongaTheProto) [inline code/datums/shuttles.dm:626]: this can be modularized

File: code/game/area/Space_Station_13_areas.dm [hunk 2]
```diff
@@ -1799,3 +1799,47 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
...
+	name = "Command-Service Hallway Bridge"
+	icon_state = "com_serv"
+
+/area/hallway/secondary/bridges/med_cargo
+	name = "Medical-Cargo Hallway Bridge"
+	icon_state = "med_cargo"
+
+/area/hallway/secondary/bridges/sci_dock
+	name = "Science-Dock Hallway Bridge"
+	icon_state = "sci_dock"
```
Comment (BongaTheProto) [inline code/game/area/Space_Station_13_areas.dm:1845]: All of this too

**Review by BongaTheProto — CHANGES_REQUESTED** [review]

File: code/game/area/Space_Station_13_areas.dm (hunk 2 above)
Comment (frontline03) [reply code/game/area/Space_Station_13_areas.dm:1845]: I do not know how to modularise

Comment (frontline03) [reply code/game/area/Space_Station_13_areas.dm:1845]: We also don't have a modular file for that

Comment (BongaTheProto) [reply code/game/area/Space_Station_13_areas.dm:1845]: We do, it's just more modularized in the area/areas folder
also it's literally just making or finding a file that emulates the path in the main code files in modular_splurt and moving all the code there.

**Review by BongaTheProto — APPROVED** [review]

**Pull request closed without merging**